config/install/core.entity_view_display.group.chat_counseling.default.yml

```diff
@@ -0,0 +1,70 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - group.type.chat_counseling
```

Comment:
This group type is not present in living_spaces.
It is added in dip_counseling/modules/dip_chat_counseling/config/install/core.entity_view_display.group.chat_counseling.default.yml